src/js/api/global-api.js

```diff
@@ -53,6 +53,9 @@ define([
     var _addPlayer = function (api) {
         _uniqueIndex++;
         api.uniqueId = _uniqueIndex;
+        api.isFirstPlayer = function() {
+            return this === _instances[0];
+        };
```

Comment:
I'm not a fan of adding API methods for internal use.  My suggestion we discussed was to make a module that exports _instances.